content/docs-v1/fundamentals.md

```diff
@@ -54,28 +55,28 @@ YAML block entries can be in any order. It doesn't matter if you write:
 ```yaml
 ---
 title: Cheatsheet
-type: page
+layout: page
 ---
 ```
 Or:
 
 ```yaml
 ---
-type: page
+layout: page
 title: Cheatsheet
 ---
 ```
 
-Certain formatting and characters (like colons within the text, or lines leading off with asterisks meant to italicize some of the text) can cause issues. In the example above, `title: My Chapter` without *My Chapter* in quotes works just fine, but more complicated cases might arise. In these cases, double quotes will help to avoid issues.
+YAML values should always be surrounded by straight quotes (" ") to ensure Quire reads the data correctly, otherwise, certain formatting and characters (like colons or asterisks) can cause issues.
 
 ```yaml
 description: "*My Chapter* is about colons :)"
 ```
 
-Anything can go within double-quotes, except for other double-quotes. If you need double-quotes, use “curly quotes”, or use a backslash to escape the double quote `\"`.
+Sometimes a caption or a chapter title will also include the use of quotations. However, two sets of straight quotes will result in an error. To avoid this, surround the caption text with straight quotes, and then use curly quotes within. To achieve this, highlight the phrase that needs curly quotes and press `Option-[`. Another option is to copy and paste them from [https://unicode-table.com/en/](https://unicode-table.com/en/).
```

Comment:
> press `Option-[`

this might be limited to certain text editors, or to only Macs. I've never really tested. Have you?